### DISCUSSION TITLE
Issues with merkle and granite multifield when used together.

### DISCUSSION THREAD

**Discussion by ridzi-tech** [merkle-open/aem-generic-multifield#66] [Q&A] [answered]
Hi team i facing an issue that if i used granite multifield and merkle multifield on saving the the data saved present in granite multifields get removed.

**Comment by michaelhoerenberg** [comment, marked as answer]
Hello @ridzi-tech,
the generic multifield will not work with the OOTB multifield.

Any component dialog definition that requires nested multifields should make use of the generic multifield only.
The generic multifield can be nested.

regards.
